APE_specutils.rst

```diff
@@ -0,0 +1,147 @@
+
+The extended packages are not detailed further here.  These are a wide range of  more domain-specific, experimental, or taste-specific packages that are likely to be desired.  Hence this tier will be more loosely structured - these will be treated as independent affiliated packages, but with a somewhat looser standard of “overlapping functionality” than in the first tier (which should have as little overlap/duplication as possible).
+
+In both tiers, Packages should aim for the same level of testing and documentation found in the `astropy` package, and should follow the standard Astropy affiliated package process to do this.  This includes, in particular, considering overlap with existing packages - this should be considered and merged when appropriate.  The Spectroscopic Coordinating Committee will assist the Astropy Coordinating Committee with this process.  Developers are encouraged to build or update existing tools to meet their needs rather than creating a new package.  
+
+The `Spectrum1D` class 
+++++++++++++++++++++++++++++++
+
+A spectroscopy package should provide a representation of the data, tools for reducing spectroscopic observations, analysis tools, and methods for interacting and visualization of the spectra.    In this section, we describe some of the functionality that will be required for a common spectroscopic package.   While equally important, we leave the specification of multi-dimensional spectral objects to future work.  At the same time, we should consider how these tools will be compatible with multi-dimensional data in the future. 
+
+A spectrum describes how the flux changes as a function of the energy.   There are a number of other parameters that can be used to describe a spectrum, and a 1D Spectrum object should have the following properties:
+
+* Dispersion
+* Flux 
+* Errors
+* Units
+* System (ie. heliocentric)
+* meta information about the provenance of the spectra
+
+To ease development, the Spectrum1D object could have energy, wavelength, and frequency properties that convert the dispersion to the appropriate dimensions.  In addition to this information, software should provide a method for reading and writing formats regularly used for spectroscopy.   Common formats include iraf, SDSS, *please list others*, midas, FITS tables, and ascii.   Reading and writing into a 1D spectrum object should use the astropy io register.   In addition, methods should exist for arithmetic on spectra1d objects including how to handle objects with different dispersion sampling.  
+
+Spectrum1D will define a specific interpretation of the meaning of a pixel coordinate value (i.e., is it the mean? Left side? Median? In what units).  This sentence is a placeholder; we plan to have a breakout session on April 21 to identify whether there is a standard we can select and finalize this choice.
```

Comment:
Just to be clear, Spectrum1D will provide a way to define the meaning of pixel coordinate, or it will actually adopt one?